src/App/Hangouts/Update.elm

```diff
@@ -1,22 +1,97 @@
 module App.Hangouts.Update exposing (..)
 
 import Navigation
+import Paginate exposing (..)
 import Routing.Route exposing (Route(..), reverse)
+import Models exposing (Model)
 import App.Hangouts.Messages exposing (Msg(..))
-import App.Hangouts.Models exposing (..)
+import App.Hangouts.Models exposing (HangoutModel)
+import App.Hangouts.Commands exposing (shuffleHangouts)
 
 
-update : Msg -> List Hangout -> ( List Hangout, Cmd Msg )
-update msg hangouts =
+update : Msg -> Model -> ( HangoutModel, Cmd Msg )
+update msg { hangoutModel, authModel } =
     case msg of
         ListHangouts ->
-            ( hangouts, Navigation.newUrl (reverse HangoutsRoute) )
+            ( hangoutModel, Navigation.newUrl (reverse HangoutsRoute) )
+
+        ShuffleHangouts ->
+            let
+                cmd =
+                    shuffleHangouts authModel.token
```

Comment:
**Suggestion** - Inline the command call and save ourselves some lines of code (`let..in` block) and a variable name. 🙂